### ISSUE TITLE
can't install node-sass

### ISSUE THREAD

**Issue by morenoh149** [keystonejs/generator-keystone#127]
ref https://github.com/keystonejs/keystone/issues/1643

**Comment by ppiekarczyk** [comment]
Does this tie into Node v4? 
The latest version of node-sass (3.3.2) now supports Node v4.

I may open up a PR to update dependencies here, they are a little behind.

**Comment by JedWatson** [comment]
@ppiekarczyk that would be great, I haven't had a chance to update & test the dependencies lately with my focus on 0.4 :)

**Comment by ppiekarczyk** [comment]
You got it!

**Comment by pathogen-** [comment]
On Windows 10 with node 4.0.0, despite updating node-sass to version 3.3.3 I still had problems with this. At different times while attempting to run the generator, and then Grunt, I was prompted to re-install both node-sass-middleware (updated to 0.9.6) and grunt-node-inspector (updated to 0.3.0). Updating these didn't work.

I then installed grunt-sass (1.0.0), because I was going to try a different Grunt build system... and suddenly everything worked! Is this because it would be creating the proper libsass bindings? I don't know!

I tried re-installing several times with the same failures and success.
